code/game/objects/items/cosmetics.dm

```diff
@@ -289,6 +289,72 @@
 		sharpness = IS_BLUNT
 		tool_behaviour = null
 
+/obj/item/comb
+	name = "comb"
+	desc = "A slick pocket comb, for cool people only."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "comb"
+	w_class = WEIGHT_CLASS_SMALL
+	force = 2
+	throwforce = 2
+	throw_speed = 6  // like card throwing
+	throw_range = 10
+	var/applied_hair = "Combover"  // incase anyone wants to make a bedhead-inator or something
+
+/obj/item/comb/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is trying to sweep their hair so fast it combusts! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	var/mob/living/carbon/human/departed = user
+	departed?.hair_style = "Balding Hair"  // oh no it burned off!
+	departed.update_hair()
+	return (FIRELOSS)
+
+
+/obj/item/comb/attack(mob/living/M, mob/living/user)
+	var/mob/living/carbon/human/patient = M
+	var/mob/living/carbon/human/barber = user
+
+	if(patient == barber)
+		return attack_self(patient)
+
+	if(barber.a_intent == INTENT_HARM || !hair_check(M, user))
+		return ..()
```

Comment:
Does this mean you just slap them if they have no hair?

Reply:
wait